internal/renderers/femtovg/lib.rs

```diff
@@ -0,0 +1,417 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
```

Comment:
There should be a crate documentation like for the other internal backend.

Reply:
Ah yes, I added docs for the main type, but not the library create. Will add.